usaspending_api/broker/management/commands/fabs_nightly_loader.py

```diff
@@ -200,18 +202,26 @@ def handle(self, *args, **options):
 
         if is_incremental_load and not submission_ids:
             logger.info("No new submissions. Exiting.")
+            return
```

Comment:
Adding this return and altering the if statement breaks the functionality of running `update_last_load_date()` when the script is an incremental run and there are no new transactions from Broker